stdlib/asm/crypto/stark/random_coin.masm

```diff
@@ -166,9 +164,9 @@ export.init_seed
     ## 4. number of auxiliary random values
     ## 5. trace length (this is already on the stack)
 
-    ## main segment width is 71 and there are 1 auxiliary segments
-    ## of width 7 using 16 random extension field elements
-    push.0x49010810
+    ## main segment width is 80 and there are 1 auxiliary segments
```

Comment:
IIUC, since we use the padded width now, the recursive verifier will be more resilient to changes in trace shape right? Basically if it stays between 72 and 80 columns, nothing (or way fewer things) would need to change right?

Reply:
Exactly! If it doesn't go to the next multiple of 8 then everything stays the same. Moreover, even when it does so, the changes should be very minimal and this should help with building the "codegen" in AirScript.